lib/Interpreter/Interpreter.cpp

```diff
@@ -1177,6 +1177,27 @@ namespace cling {
   }
 
   void Interpreter::unload(Transaction& T) {
+    // Clear any stored states that reference the llvm::Module.
+    // Do it first in case
```

Comment:
I guess it's a euphemism, generally bad things, like "just in case"
Specifically, do it first while we know Transaction& T to be valid.